app/src/main/res/values/strings.xml

```diff
@@ -38,9 +38,9 @@
     <string name="title_activity_calibration_override">Calibration Override</string>
     <string name="pref_header_cloud_storage">Cloud Storage</string>
     <string name="auto_configure_title">Auto configure</string>
-    <string name="prefs_auto_config_summary">Auto configure using a barcode.</string>
-    <string name="scan_share2_barcode">Scan Share Barcode</string>
-    <string name="pref_share2_scan_barcode_summary">Or scan the barcode on the share receiver</string>
+    <string name="prefs_auto_config_summary">Auto configure using a QR code.</string>
```

Comment:
I don't think we are limited to QR codes with the auto-configuration feature so I wouldn't think this would be good to change.

Reply:
@jamorham Hmm, I currently don't now where the barcodes should came from. As far as I know, xDrip only provides a QR code if "Show Settings QR codes" is selected. Please can you clarify, when and how a barcode is created/displayed?

@Navid200 Do you know of any real barcode in xDrip?

Reply:
I don't.  But, there is a lot I don't know.

It may be best to break this up into two different PRs.
I already have this (see image) in the [documentation](https://navid200.github.io/xDrip/docs/Legend.html).

![Screenshot 2022-02-23 161230](https://user-images.githubusercontent.com/51497406/155409297-d7f18d3d-cfec-475f-bbd7-a25e51ef1b03.png)

If we/I open a PR with only that, it may pass.  Then, the conversation about the bar code can continue and may resolve also.

Reply:
Splitting off the QR change seems like overkill. I looked into the scanner the other day as part of a separate discussion:
https://github.com/NightscoutFoundation/xDrip/discussions/1991#discussioncomment-2237437
The **TL;DR** is that there are two separate scanners in the UI, and the proposed alterations affect the one that _was_ intended for QR codes.

I support the QR-related changes in this PR, especially since they don't replace the labels for BOTH scanners.
At this point in time, "QR scanner" is probably the more relevant/user-friendly label, even if it _can_ secretly do legacy barcodes under the hood.